README.md

```diff
@@ -1 +1,25 @@
 # hw01
+
+## About Me
+My name is Ming Cui. I received my master's degree in business administration from University of Science and Technology of China. I am working on my master's in social sciences at UChicago.
+
+## Why this course?
+**Computational tools** are trending. You may find
+
+1. Computational Neuroscience
+1. Computational Biology
+1. Computational Sociology
+1. Computational Psychology
+1. Computational Economics
```

Comment:
Your list is very clear and easy to read, maybe you can experiment with adding sub items next time.